blueocean-dashboard/package.json

```diff
@@ -66,13 +70,8 @@
     "react-autocomplete": "1.4.0",
     "react-dom": "15.4.2",
     "react-i18next": "1.10.1",
-    "react-redux": "4.4.5",
-    "react-remarkable": "1.1.1",
     "react-router": "2.8.1",
-    "recharts": "1.0.0-alpha.4",
-    "redux": "3.6.0",
-    "redux-thunk": "2.1.0",
-    "reselect": "2.5.4"
+    "recharts": "1.0.0-alpha.4"
```

Comment:
why?

Reply:
@scherler not actually an addition https://github.com/jenkinsci/blueocean-plugin/pull/1627/files/e0da8c181b8590b5716b3d7815c4f6b52a1daade#diff-0ca33611a0c8a93fe297afbfc1b1a8aaL72 has it a few lines above. Just a case of git diff getting confused